### ISSUE TITLE
BUG: Chat Report shown instead of room title when created offline

### ISSUE THREAD

**Issue by neil-marcellini** [Expensify/App#12629]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Sign in with any account
2. Create a workspace if needed
3. Go offline
4. Create a room with a specific name

## Expected Result:
You are navigated to the room you created and it has the correct name in the header, and it's grayed out.

## Actual Result:
You are navigated to the room you created but the name is "Chat Report" in the header.

## Workaround:
Ignore it and go online

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** v1.2.26-0
**Reproducible in staging?:** N
**Reproducible in production?:** N
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:**
Note that this is currently only on main, but it will be on staging soon.
https://user-images.githubusercontent.com/26260477/200973271-17ae2f42-961f-419e-99d9-df4d46cbf6ea.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @neil-marcellini 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1668041691611749

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by miljakljajic** [comment]
I wasn't able to reproduce this, @neil-marcellini - it's working as expected for me, both offline and online: 

<img width="1133" alt="image" src="https://user-images.githubusercontent.com/28715117/201975627-edbb30c1-eb29-4b2d-ab8e-e06073756f6a.png">

**Comment by neil-marcellini** [comment]
That's odd, it still doesn't work for me. I tried on multiple accounts.

https://user-images.githubusercontent.com/26260477/201980529-d45ea5c8-d0c7-499e-9303-61dca0031e97.mov

**Comment by thesahindia** [comment]
Same, reproducible for me as well!

**Comment by miljakljajic** [comment]
Ah, strange! @neil-marcellini, do you believe this can be worked on by an external contributor?

**Comment by neil-marcellini** [comment]
I will add some test steps to that draft PR and post screen recordings, and then put it up for review. This bug is a regression created in [this PR](https://github.com/Expensify/App/pull/12157).

**Comment by roryabraham** [comment]
Ah, nice job @neil-marcellini 👏🏼

**Comment by neil-marcellini** [comment]
I'll finish testing and get that PR up for review.

**Comment by neil-marcellini** [comment]
For some reason there's a jest unit test failing on my PR so I'll need to look into that tomorrow.

**Comment by neil-marcellini** [comment]
I ended up closing my PR. See my comment here https://github.com/Expensify/App/pull/12874#issuecomment-1325563913. Here's the checklist, let's close this out.

**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@miljakljajic / @neil-marcellini] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@miljakljajic / @neil-marcellini] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@miljakljajic / @neil-marcellini] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@miljakljajic / @neil-marcellini] A [regression test](https://expensify.testrail.io/index.php?/dashboard) has been added or updated so that the same bug will not reach production again. Link to the GH issue for creating the test here:

**Comment by neil-marcellini** [comment]
@miljakljajic would you please take care of the last two steps?

**Comment by neil-marcellini** [comment]
Actually to avoid more duplication let's have this issue https://github.com/Expensify/App/issues/12883 complete the checklist. I will comment over there letting them know that the first two steps are already done.
